src/Weikio.ApiFramework.SDK.DatabasePlugin/CodeGeneration/DtoBase.cs

```diff
@@ -4,8 +4,8 @@ public class DtoBase
     {
         public object this[string propertyName]
         {
-            get { return GetType().GetProperty(propertyName).GetValue(this, null); }
-            set { GetType().GetProperty(propertyName).SetValue(this, value, null); }
+            get { return GetType().GetProperty(propertyName.Replace("@", string.Empty)).GetValue(this, null); }
```

Comment:
Removing the @ fixes the issue that build SQL is trying to query @class field, which does not exist. The field name is class.

Reply:
Would it be safer to use `.TrimStart('@')` instead of `Replace`?